docs/ru/interfaces/third-party/integrations.md

```diff
@@ -77,6 +77,7 @@
 - C#
     - [ADO.NET](https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/ado-net-overview)
         - [ClickHouse.Ado](https://github.com/killwort/ClickHouse-Net)
+        - [ClickHouse.Client](https://github.com/DarkWanderer/ClickHouse.Client)
```

Comment:
other languages probably also need updating

Reply:
I can only speak for C# :)